README.md

```diff
@@ -80,7 +80,7 @@ Make sure the main activity in your AndroidManifest extends `android.support.v7.
           };
   LiveEngage.getInstance().setUserProfileValues(chatProfile);
   ```
-7. Optional: Add a JWT token for oAuth support when starting a conversation. Make sure you configure the Live Person data source to support the oAuth 2 authentication in terms of Live person public keys, Signing identities and possible custom claims definitions.
+7. Optional: Add a JWT token for oAuth support when starting a conversation. Make sure you configure the Live Person data source to support the oAuth 2 authentication in terms of Live person public keys, Signing identities and possible (custom) claims definitions.
```

Comment:
I think it's good to mention that `setAuthenticationCode` should be called before `showChat`.